README.md

```diff
@@ -46,27 +46,68 @@ The snapshot role supports sets of volumes.  Sets may contain any number of volu
 Sets are defined in the following format:
 
 ```text
-    snapshot_lvm_set:
+    snapshot_test_set:
       name: snapset1
       volumes:
         - name: snapshot VG1 LV1
           vg: test_vg1
           lv: lv1
-          percent_space_required: 20
-        - name: snapshot VG1 LV1
+          percent_space_required: 15
+          mountpoint: /mnt/lv1_mp
+        - name: snapshot VG2 LV3
+          vg: test_vg2
+          lv: lv3
+          mountpoint: /mnt/lv3_mp
+          percent_space_required: 15
+        - name: snapshot VG2 LV4
           vg: test_vg2
+          lv: lv4
+          mountpoint: /mnt/lv4_mp
+          percent_space_required: 15
+        - name: snapshot VG3 LV7
+          vg: test_vg3
+          lv: lv7
+          mountpoint: /mnt/lv7_mp
+          percent_space_required: 15
+```
+
+If the user is indicating the mount action and wants the origin of the snapshot to be mounted rather
+than the snapshot, the set would be defined with the additional field "mount_origin" for any entry
+that it applies, for example:
+
+```text
```

Comment:
```suggestion
```yaml
```

Please use the appropriate highlighter, this will be then highlighted pretty in README.html, in GItHub preview, in Galaxy and AH.

Reply:
In other places where it applies too, please

Reply:
fixed.  thanks.